### PR TITLE
fix(editor): Add workflow scopes when initializing workflow 

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

image

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/PAY-1858/bug-node-popup-contains-disabled-options

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Review by mutdmour — CHANGES_REQUESTED** [review]
Can you please add tests for this bug?

File: packages/editor-ui/src/stores/workflows.store.ts [hunk 1]
```diff
@@ -699,6 +699,10 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, () => {
 		};
 	}
 
+	function setWorkflowScopes(scopes: IWorkflowDb['scopes']): void {
+		workflow.value.scopes = scopes;
```
Comment (mutdmour) [inline packages/editor-ui/src/stores/workflows.store.ts:703]: do all workflows have scopes? Even old workflows?

**Review by MarcL — COMMENTED** [review]
Can we add some tests for the scopes here?

File: packages/editor-ui/src/stores/workflows.store.ts (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/stores/workflows.store.ts:703]: nvm I see it now returned for all workflows

**Review by mutdmour — DISMISSED** [review]
Thanks for adding tests. LGTM. Though would love to have e2e tests for this feature (context menu) but we can handle that separately

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.55.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.55.2)

**Pull request merged**